README.md

```diff
@@ -349,7 +349,7 @@ rum.realm=<realm>
 rum.access.token=<a valid Splunk RUM access token for the realm>
 ```
 
-# License and versioning
+# License
```

Comment:
License and versioning is used in multiple splunk otel projects: [splunk-otel-java](https://github.com/signalfx/splunk-otel-java), [splunk-otel-js](https://github.com/signalfx/splunk-otel-js), [splunk-otel-python](https://github.com/signalfx/splunk-otel-python), [splunk-otel-js-web](https://github.com/signalfx/splunk-otel-js-web), could be more

Reply:
I can create more PRs on Monday